cmake/kokkos_tribits.cmake

```diff
@@ -366,15 +367,16 @@ FUNCTION(KOKKOS_SET_LIBRARY_PROPERTIES LIBRARY_NAME)
       ${LIBRARY_NAME} PUBLIC ${KOKKOS_LINK_OPTIONS}
     )
   ENDIF()
+  ENDIF()
```

Comment:
```suggestion
  ENDIF()
```

Reply:
Is it OK to ignore all link options here? Do we never set anything that could be useful for a different language?

Reply:
the problem is that we can't use CUDA as linker language etc. i.e. this language stuff only really works if you don't need linker options ...

Reply:
If this ever changes or we need to handle that we will do it then.